x/bank/client/cli/broadcast.go

```diff
@@ -0,0 +1,46 @@
+package cli
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	"github.com/spf13/cobra"
+	amino "github.com/tendermint/go-amino"
+)
+
+// GetSignCommand returns the sign command
+func GetBroadcastCommand(codec *amino.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "broadcast <file>",
```

Comment:
Maybe change it to `broadcast-tx`? CC: @jackzampolin @alexanderbez thoughts ?

Reply:
In that case, we should rename `sign` to `sign-tx` for consistency's sake

Reply:
I prefer `/broadcast` and `/sign` but thats just me. Maybe `/tx/broadcast` and `/tx/sign` but not with dashes like that

Reply:
Amended, the endpoints are now under `/tx/sign` and `/tx/broadcast`

Reply:
Why pass a filename instead of reading from stdin? If you read from stdin, then this can be piped to directly (or stdin redirected from a file)?